### ISSUE TITLE
Instructions install Elm-0.12.3 and elm-debugger can't resolve dependencies

### ISSUE THREAD

**Issue by jcollard** [elm-lang/debug.elm-lang.org#5]
```
git clone https://github.com/elm-lang/Elm.git
cd Elm
git checkout debugger
cabal install
```

Results in `Installed Elm-0.12.3`

Then running:

```
git clone https://github.com/elm-lang/elm-debugger.git
cd elm-debugger
cabal configure
```

```
Resolving dependencies...
Configuring elm-debugger-0.1...
setup: At least the following dependencies are missing:
Elm >=0.13
```

**Comment by PedroSena** [comment]
Same here

**Comment by jvoigtlaender** [comment]
@PedroSena, you are trying to install Elm-0.12.3? Note that current Elm version is 0.15.1. Instructions for installing it are at http://elm-lang.org/install. Something like `git clone https://github.com/elm-lang/Elm.git` doesn't feature there. Maybe you are trying to follow far outdated install instructions?

**Comment by jvoigtlaender** [comment]
Actually, are you maybe looking for https://github.com/elm-lang/elm-reactor? That comes with installing the recent Elm platform.

@evancz, maybe this repo here should point people to the `elm-reactor`? Does it make sense these days to try to install something from this repo here at all?

**Comment by PedroSena** [comment]
Hi @jvoigtlaender I just tried to follow the instructions on the main README file on this repository.
My Elm version is indeed 0.15.1 however I do get the exact same error message as the other user when I try to configure the elm-debugger.
ELM_HOME is properly set and elm executable is also in my PATH so I believe the issue is somewhere else.
I had the impression that the debugger was separated from the reactor from the video where I saw it, maybe that caused the confusion.
Thanks
